src/theory/datatypes/datatypes_rewriter.h

```diff
@@ -51,6 +51,28 @@ struct SygusSymBreakOkAttributeId
 typedef expr::Attribute<SygusSymBreakOkAttributeId, bool>
     SygusSymBreakOkAttribute;
 
+/** sygus var free
+ *
+ * This attribute is used to mark whether sygus operators have free occurrences
+ * of variables from the formal argument list of the function-to-synthesize.
```

Comment:
why "formal argument list" rather than "argument list"?

Reply:
"Formal argument list" means the expected arguments in the declaration of the function.

Reply:
What I don't understand is the need for "formal".

Reply:
This is standard terminology?